megalinter/descriptors/xml.megalinter-descriptor.yml

```diff
@@ -8,15 +8,25 @@ file_extensions:
   - ".xml"
 linters:
   # XML LINT
-  - linter_name: xmllint
+  - class: XmlLintLinter
+    linter_name: xmllint
+    variables:
+      - name: XML_XMLLINT_AUTOFORMAT
+        default_value: "false"
+        description: If set to `true`, it will reformat and reindent the output
+      - name: XML_XMLLINT_INDENT
+        default_value: "  "
+        description: The number of indentation spaces when `XML_XMLLINT_AUTOFORMAT` is `true`
     linter_url: http://xmlsoft.org/xmllint.html
     linter_rules_url: http://xmlsoft.org/xmllint.html#diagnostics
     version_extract_regex: "(?<=libxml version )\\d+(\\d+)+"
     cli_lint_mode: list_of_files
+    cli_lint_fix_arg_name: "--format"
```

Comment:
When this is not the real argument string, use the same dummy one than for others so it won't be added in the command :)
  `  cli_lint_fix_arg_name: "--megalinter-fix-flag" # Workaround for MegaLinter `

Reply:
As you have already deduced, the `--format` can only be used in "file" mode because as I have put in the first message of this PR, it expects an `--output` parameter which evidently has to be file to file.

Why do you say that `--format` is not the actual argument? It is:

![image](https://user-images.githubusercontent.com/950602/211663405-09ae1815-a703-4d27-b9e0-387d635025f2.png)

![image](https://user-images.githubusercontent.com/950602/211663581-4b018b87-2300-4b49-af6d-f2e0ca34bf4d.png)

Reply:
ok i didn't know it was, i thought it was managed with another env var